typings/index.d.ts

```diff
@@ -13,7 +13,7 @@ import {
   TimestampStylesString,
   underscore,
 } from '@discordjs/builders';
-import BaseCollection from '@discordjs/collection';
+import Collection from '@discordjs/collection';
```

Comment:
For consistency with the JS-side:
```suggestion
import { Collection } from '@discordjs/collection';
```